src/carousel/carousel.ts

```diff
@@ -163,10 +196,27 @@ export class NgbCarousel implements AfterContentChecked,
     this.wrap = config.wrap;
     this.keyboard = config.keyboard;
     this.pauseOnHover = config.pauseOnHover;
+    this.pauseOnFocus = config.pauseOnFocus;
     this.showNavigationArrows = config.showNavigationArrows;
     this.showNavigationIndicators = config.showNavigationIndicators;
   }
 
+  @HostListener('keydown.arrowLeft')
```

Comment:
We don't use `@HostListener()` on the project. Would you mind moving all of them in the top `@Component()` declaration ?

Reply:
I just realised we missed some other in the file on the previous PR. Please, take the opportunity to move _all_ of them

Reply:
Thank you for your comment. I have added a commit to change this.